streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java

```diff
@@ -15,15 +15,17 @@
  * limitations under the License.
  */
 
-package org.apache.kafka.streaming.processor;
+package org.apache.kafka.streams.processor;
 
 import org.apache.kafka.common.KafkaException;
 import org.apache.kafka.common.serialization.Deserializer;
 import org.apache.kafka.common.serialization.Serializer;
-import org.apache.kafka.streaming.processor.internals.ProcessorNode;
-import org.apache.kafka.streaming.processor.internals.ProcessorTopology;
-import org.apache.kafka.streaming.processor.internals.SinkNode;
-import org.apache.kafka.streaming.processor.internals.SourceNode;
+import org.apache.kafka.streams.KafkaStreaming;
```

Comment:
Doesn't the style checker fail due to this? It seems the style checker runs only when you run tests. KafkaStreaming is used only in JavaDoc. There is not reference to KafkaStreaming in the actual code. And the style checker complains that this is an unused import.